frontend/src/components/scheduleEntry/DialogActivityEdit.vue

```diff
@@ -22,8 +23,8 @@
 </template>
 
 <script>
-import DialogForm from './DialogForm.vue'
-import DialogBase from './DialogBase.vue'
+import DialogForm from '@/components/dialog/DialogForm.vue'
```

Comment:
Wird hier absichtlich von relativem zu absolutem pfad gewechselt?
Wann soll man relative und wann absolute pfade verwenden?

Reply:
#2389

Reply:
Ja war absichtlich. Ich hab die spezifischen ActivityDialogs verschoben in den Ordner "scheduleEntry", damit dies näher ist bei dem restlichen Inhalt wo es auch verwendet wird (haben wir ähnlich schon so für "material"). Unter "dialog" möchte ich langfristig eigentlich nur die Basiskomponenten drin haben, analog zu "form".

Relative Pfade finde ich ok für den gleichen Ordner oder einen Unterordner (z.B. referenzieren einer Subkomponente welche eindeutig zur aktuellen Komponente gehört). Ich würde aber nicht damit starten, auf übergeordnete  Ordner mit relativen Pfaden zu operieren (e.g. `./../../`). Das macht Refactorings unnötig schwer.